opencga-catalog/src/main/java/org/opencb/opencga/catalog/db/mongodb/MetaMongoDBAdaptor.java

```diff
@@ -221,20 +230,14 @@ public OpenCGAResult updateJWTParameters(ObjectMap params) throws CatalogDBExcep
 
         Document adminDocument = new Document();
         if (StringUtils.isNotEmpty(params.getString(SECRET_KEY))) {
+            // Ensure JWT secret key is long and secure
+            String secretKey = params.getString(SECRET_KEY);
+            if (!PasswordUtils.isStrongPassword(secretKey, 30)) {
```

Comment:
you should use here the SECRET_KEY_MIN_LENGTH

Reply:
fixed